internal/cri/types/labels.go

```diff
@@ -1,3 +1,18 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
```

Comment:
Since we are using only the constants from this file, is it needed to maintain the complete copy of  lables.go?

Reply:
left the copyright and it shows as a move

Reply:
all the labels looked interesting to me...

Reply:
also would like to move these constants/helpers to the cri-api